bin/git-summary

```diff
@@ -189,39 +190,56 @@ uncommitted_changes_count() {
   git status --porcelain | wc -l
 }
 
-# summary
-if [ -n "$SUMMARY_BY_LINE" ] && [ -n "$SUMMARY_ONELINE" ]; then
-  echo "$project / lines: $(line_count "${paths[@]}")"
-elif [ -n "$SUMMARY_BY_LINE" ]; then
-  echo
-  echo " project     : $project"
-  echo " lines       : $(line_count "${paths[@]}")"
-  echo " authors     :"
-  lines "${paths[@]}" | sort | uniq -c | sort -rn | format_authors
-elif [ -n "$SUMMARY_ONELINE" ]; then
-  echo "$project / age: $(repository_age) / last active: $(last_active) / active on $(active_days $commit) days / commits: $(commit_count $commit) / uncommitted: $(uncommitted_changes_count)"
-else
-  echo
-  echo " project     : $project"
-  echo " repo age    : $(repository_age)"
-  echo " last active : $(last_active)"
-  # shellcheck disable=SC2086
-  echo " active on   : $(active_days $commit) days"
-  # shellcheck disable=SC2086
-  echo " commits     : $(commit_count $commit)"
-
-  # The file count doesn't support passing a git ref so ignore it if a ref is given
-  if [ "$commit" == "HEAD" ]; then
-    echo " files       : $(file_count)"
+print_summary_by_line() {
+  if [ "$OUTPUT_STYLE" == "tabular" ]; then
+    tabular_headers="# Repo | Lines"
+    echo -e "$tabular_headers\n$project | $(line_count "${paths[@]}")" | column -t -s "|" -o "|"
```

Comment:
When running this, I get:
```
column: invalid option -- 'o'
usage: column [-txne] [-c columns] [-s sep] [file ...]
```

Reply:
Hm, that is weired. What OS and do you use and what version of `column` do you have. According to https://man7.org/linux/man-pages/man1/column.1.html the `-o` option is there and on my ubuntu 22.04 it's also working.

Reply:
`man column` show that:
```
HISTORY
     The column command appeared in 4.3BSD-Reno.
```

There is not version output option in my column

Reply:
Double checked. The BSD column command just don't container the `-o` option. I have to think about this how to solve it because the `-o` would be quite important to have a separator between columns, else my overall-command can't distinguish the single columns...

Reply:
I got rid of the `-o` option.